### PR TITLE
gh-86388 Remove deprecated behaviors in randrange()

### PR DESCRIPTION
**Pull request by rhettinger**

### CODE REVIEW

File: Lib/random.py [hunk 1]
```diff
@@ -282,67 +282,34 @@ def randbytes(self, n):
...
-                  'since Python 3.10 and will be removed in a subsequent '
-                  'version',
-                  DeprecationWarning, 2)
+        istep = _index(step)
         # Fast path.
         if istep == 1:
             if width > 0:
                 return istart + self._randbelow(width)
-            raise ValueError("empty range for randrange() (%d, %d, %d)" % (istart, istop, width))
+            raise ValueError(f"empty range in randrange({start}, {stop}, {step})")
```
Comment (serhiy-storchaka) [inline Lib/random.py]: Ьфниу огіе 'randrange({start}, {stop})'?

File: Doc/whatsnew/3.12.rst [hunk 2]
```diff
@@ -102,7 +102,11 @@ Deprecated
 Removed
 =======
 
-
+* Removed randrange() functionality deprecated since Python 3.10.  Formerly,
```
Comment (serhiy-storchaka) [inline Doc/whatsnew/3.12.rst]: I think it would be better to move to section "Changes in the Python API".

Comment (rhettinger) [reply Doc/whatsnew/3.12.rst]: Okay.

File: Lib/random.py (hunk 1 above)
Comment (rhettinger) [reply Lib/random.py]: Okay.

**Pull request merged**